.buildkite/cpu_scaling_pipeline/generate_pipeline.jl

```diff
@@ -62,8 +90,17 @@ pipeline = Dict(
         "wait",
         Dict(
             "group" => "Moist Baroclinic Wave, weak scaling",
-            "steps" => [generate_step.(nodes, helems)...],
+            "steps" => [
+                generate_step_ws.(ws_nodes, ws_helems, procspernode)...,
+            ],
+        ),
+        Dict(
+            "group" => "Moist Baroclinic Wave, strong scaling",
+            "steps" => [
+                generate_step_ss.(ss_nodes, ss_helems, procspernode)...,
+            ],
         ),
     ],
 )
-YAML.write_file(".buildkite/cpu_scaling_pipeline/pipeline.yml", pipeline)
+#YAML.write_file(".buildkite/cpu_scaling_pipeline/pipeline.yml", pipeline)
+YAML.write_file("pipeline.yml", pipeline)
```

Comment:
Is there a reason the output path has been changed?

Reply:
No. I reverted that change.